cli.js

```diff
@@ -28,56 +30,68 @@ Usage:
 Options:
     --headless      No window, no video. Also disables --artwork. Default: false
 \n`)
-	process.exit(0)
+    process.exit(0)
 }
 
 if (argv.version || argv.v) {
-	process.stdout.write(`fasp-server v${pkg.version}\n`)
-	process.exit(0)
+    process.stdout.write(`fasp-server v${pkg.version}\n`)
+    process.exit(0)
 }
 
 const showError = (err) => {
-	if (process.env.NODE_DEBUG === pkg.name) console.error(err)
-	else console.error(err.message || (err + ''))
-	process.exit(1)
+    if (process.env.NODE_DEBUG === pkg.name) console.error(err)
+    else console.error(err.message || (err + ''))
+    process.exit(1)
+}
+
+/**
+ * Returns a promise that resolves to a valid port number
+ * @param {number} [preferredPort] - The port that is preferred
+ */
+const ensurePort = (preferredPort) => {
+    if (!Number.isNaN(preferredPort) && preferredPort > 0) return Promise.resolve(preferredPort)
+    return getPort()
 }
 
 const fs = require('fs')
 
 const cmd = argv._[0]
 if (cmd === 'init') {
-	const mkdirp = require('mkdirp')
-	const {randomBytes} = require('crypto')
+    const mkdirp = require('mkdirp')
```

Comment:
Use tabs as specified in the `.editorconfig` file.

Reply:
Done